SmartDeviceLink/SDLIAPControlSession.h

```diff
@@ -0,0 +1,64 @@
+//
+//  SDLIAPControlSession.h
+//  SmartDeviceLink
+//
+//  Created by Nicole on 4/16/19.
+//  Copyright © 2019 smartdevicelink. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "SDLIAPSessionDelegate.h"
+
+@class EAAccessory;
+@class SDLIAPSession;
+
+NS_ASSUME_NONNULL_BEGIN
+
+typedef void (^SDLIAPControlSessionRetryCompletionHandler)(void);
+typedef void (^SDLIAPControlSessionCreateDataSessionCompletionHandler)(EAAccessory *connectedaccessory, NSString *indexedProtocolString);
+
+/**
+ *  A control session is used by to get the unique protocol string needed to create a session with Core. The control session is only used on legacy head units that do not support the multisession protocol which allows multiple apps to connect over the same protocol string. When the protocol string is received from Core, the control session is closed as a new session with Core must be established with the received protocol string. Core has ~10 seconds to send the protocol string, otherwise the control session is closed and new attempt is made to establish a control session with Core.
+ */
+@interface SDLIAPControlSession : NSObject
```

Comment:
`SDLIAPSession` class should be removed

Reply:
The `@class SDLIAPSession` import?

Reply:
Sorry, I thought the `SDLIAPSession` class was now unused